paymentsheet/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt

```diff
@@ -267,19 +267,7 @@ internal class DefaultLinkAccountManager @Inject constructor(
                     consumerSessionClientSecret = account.clientSecret,
                     consumerPublishableKey = account.consumerPublishableKey.takeIf { !config.passthroughModeEnabled },
                     active = config.passthroughModeEnabled,
-                ).mapCatching {
-                    if (config.passthroughModeEnabled) {
-                        linkRepository.shareCardPaymentDetails(
```

Comment:
👀 Extracted to `shareCardPaymentDetails()` below. Removed the `passthroughModeEnabled` check to let the caller decide.